openpype/pipeline/workfile/build_workfile.py

```diff
@@ -236,9 +240,14 @@ def get_build_presets(self, task_name, asset_doc):
             Dict[str, Any]: preset per entered task name
         """
 
-        host_name = os.environ["AVALON_APP"]
+        from openpype.pipeline.context_tools import (
+            get_current_host_name,
+            get_current_project_name,
+        )
```

Comment:
Any reason why these imports can't be at the top of the file?

Reply:
Causing cross import bug.